ksql-api/src/main/java/io/confluent/ksql/api/server/ApiServerConfig.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.server;
+
+import io.confluent.common.config.AbstractConfig;
+import io.confluent.common.config.ConfigDef;
+import io.confluent.common.config.ConfigDef.Importance;
+import io.confluent.common.config.ConfigDef.Type;
+import io.confluent.ksql.util.KsqlConfig;
+import io.vertx.core.json.JsonObject;
+import java.util.Map;
+
+/**
+ * Config for the API server
+ */
+public class ApiServerConfig extends AbstractConfig {
+
+  private static final String PROPERTY_PREFIX = "apiserver.";
```

Comment:
Can we add  a `ksql.` prefix to this please?

Reply:
It's a little bit further down in the file :)